### PR TITLE
papercut: persist rules.json

### PR DESCRIPTION
**Pull request by abejenaru**

### CODE REVIEW

File: bucket/papercut.json [hunk 1]
```diff
@@ -5,11 +5,17 @@
     "license": "Apache-2.0",
     "url": "https://github.com/ChangemakerStudios/Papercut/releases/download/5.5.1/Papercut.5.5.1.zip",
     "hash": "ec67c13414057cf9bf4e08002bc39ba148e2e79c7ec1d321a599f05374f7f26d",
+    "pre_install": [
```
Comment (Ash258) [inline bucket/papercut.json]:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\rules.json\")) { Set-Content \"$dir\\rules.json\" '[]' -Encoding Ascii }",
```

File: bucket/papercut.json [hunk 2]
```diff
@@ -5,11 +5,17 @@
...
+         "    New-Item -Force -Path \"$dir\" -Name $file -Value '[]' | Out-Null",
+        "}"
+    ],
     "persist": [
         "Incoming",
-        "Logs"
+        "Logs",
+        "rules.json"
     ],
-    "bin": "Papercut.exe",
```
Comment (Ash258) [inline bucket/papercut.json]: Why was bin removed?

Comment (abejenaru) [reply bucket/papercut.json]: It'a s GUI app - the shortcut is enough.

**Comment by abejenaru** [comment]
> Link to documentation where is this configuration file mentioned?

From application log: "Attempting to Load Rules from "C:\Users\XXX\scoop\apps\papercut\current\rules.json"

File: bucket/papercut.json (hunk 2 above)
Comment (Ash258) [reply bucket/papercut.json]: I do not see a reason to remove binary. There are lots of manifests with applications which are graphical only and have binary.

Comment (abejenaru) [reply bucket/papercut.json]: In this case, launching the shim gives an empty console window that stays on :(
I can't see any benefit from keeping the shim.

**Pull request merged**